src/input/datetime.rs

```diff
@@ -390,10 +392,12 @@ pub fn int_as_datetime<'py>(
     match DateTime::from_timestamp_with_config(
         timestamp,
         timestamp_microseconds,
-        &TimeConfig {
-            unix_timestamp_offset: Some(0),
-            ..Default::default()
-        },
+        &DateTimeConfig::builder()
+            .time_config(TimeConfig {
```

Comment:
⛏️ Feel like we should use the builder here, but just left as is for now to make this a simple update